app/models/assignment_repo/creator.rb

```diff
@@ -116,7 +116,7 @@ def add_user_to_repository!(github_repository_id)
       options = {}.tap { |opt| opt[:permission] = "admin" if assignment.students_are_repo_admins? }
 
       github_repository = GitHubRepository.new(organization.github_client, github_repository_id)
-      invitation = github_repository.invite(user.github_user.login_no_cache, options)
```

Comment:
`(use_cache: false)` is the new API for skipping local + GitHub caches. I'll be removing the definitions for these `_no_cache` methods if we're able to convert all github_ resources to the new API.